ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.18f1
-m_EditorVersionWithRevision: 2020.3.18f1 (68f137dc9bbe)
+m_EditorVersion: 2020.3.19f1
```

Comment:
Change the version back to 18f1. I thought it was 19f1 too but Dennis made a post https://moodle.socs.uoguelph.ca/mod/forum/discuss.php?d=3988